setup.py

```diff
@@ -26,6 +26,7 @@
 
 
 dev_requires = [
+    "black",
     "ruff",
```

Comment:
We should presumably pin these to the same versions as in pre-commit so we ensure the same formatting